lib/eco/export/jruby.py

```diff
@@ -365,49 +367,74 @@ def annotate_tree(self, src_file_name, log_file_name):
 
         # Process graph of reachable objects and annotate parse tree.
         self.remove_all_annotations()
+
         for obj in reachable_objects:
-            if (isinstance(obj, Method) and obj.source.file != "(unknown)"
-                and not obj.is_hidden() and not obj.is_core()
-                and not obj.is_library() and obj.source.file != "(eval)"
-                and obj.name != "<main>"):
+            if (isinstance(obj, Method) and obj.source.file != "(unknown)" and
```

Comment:
Not a fixable comment in this PR, but I can't help feel that `obj.source.file` being "(unknown)" isn't a great way of signifying a special status!